pkg/sql/ambiguous_commit_test.go

```diff
@@ -92,7 +92,9 @@ func TestAmbiguousCommit(t *testing.T) {
 						sendNext: func(ctx context.Context, done chan<- kv.BatchCall) {
 							if ambiguousSuccess {
 								interceptDone := make(chan kv.BatchCall)
-								transport.SendNext(ctx, interceptDone)
+								go func() {
```

Comment:
no need for the extra function here, this is equivalent to `go transport.SendNext(ctx, interceptDone)`